drivers/Makefile.include

```diff
@@ -82,6 +82,10 @@ ifneq (,$(filter fxos8700,$(USEMODULE)))
   USEMODULE_INCLUDES += $(RIOTBASE)/drivers/fxos8700/include
 endif
 
+ifneq (,$(filter extend_gpio,$(USEMODULE)))
```

Comment:
Should probably be
```suggestion
ifneq (,$(filter extend,$(USEMODULE)))
```
to address any `extend_*` module.

Reply:
Fixed, and made a bit more alphabetical.